drupal/.gitignore

```diff
@@ -25,21 +25,4 @@ web/_ping.php
 # Ignore files generated by PhpStorm
 /.idea/
 
-# Ignore files generated by scaffold
```

Comment:
We should still ignore files that we don't actually need instead of adding unnecessary files to the repo.

Reply:
Updated ignore files